### DISCUSSION TITLE
The check-in at the end of the rainbow 🌈  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#49641] [New to GitHub]
It may not be a pot of gold, but this check-in has plenty to treasure! 

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @fabianorauzer-egsys: Has been helpful all around the community, but we were especially impressed by their clear step-by-step instructions and ways to learn more in [this discussion post](https://github.com/orgs/community/discussions/49352). 
*  @dhiant: is getting the message that their repository `been retired and cannot be reused`- a common situation - be a hero and see if you can [help them](https://github.com/orgs/community/discussions/48066). 

📚 **Rousing Reads**: GitHub blog goodies 🍭
* **A beginner’s guide to learning to code with GitHub Codespaces**: When you’re new to coding, it’s easy to get stuck completing endless tutorials. You can apply what you’ve learned (and learn even more) through GitHub Codespaces, get started with our [guide](https://github.blog/2023-02-22-a-beginners-guide-to-learning-to-code-with-github-codespaces/). 
* **Release Radar · February 2023 Edition**: The latest big releases in the open source world hand-picked by @mishmanners! Take a [read](https://github.blog/2023-03-05-release-radar-feb-2023/) and let us know your favorite.
* **Why Python keeps growing, explained**: @blackgirlbytes dives into why more people are using Python 🐍 than ever, its key use cases, and why it’s still so popular 30-plus years after it was first released in her [blogpost](https://github.blog/2023-03-02-why-python-keeps-growing-explained/).

🆒 **Cool stuff™️** happening around GitHub and the Community
* **Add your pronouns to your GitHub profile**: View the changelog [here](https://github.blog/changelog/2023-03-01-add-pronouns-to-your-github-profile/). 
* **Temporary Interaction Limits on New Accounts**: In an effort to reduce spammy behavior, we are instituting temporary interaction limits to bar accounts less than 24hrs-old from participating in the github.com/community Discussions space. [Read the notice](https://github.com/orgs/community/discussions/48334). 
* **SMS and TOTP can now both be registered 2FA methods**: [Learn more](https://github.blog/changelog/2023-03-02-sms-and-totp-can-now-both-be-registered-2fa-methods/) about how you can register them and keep your account secure 🔒.
Accessibility improvements for the contribution graph: The contribution graph 🟩 now supports keyboard interaction and compatibility with assistive technologies. [Find out the details](https://github.blog/changelog/2023-03-02-accessibility-improvements-for-the-contribution-graph/). 

Feeling lucky 🍀? Share what projects you’re working on with us.

**Comment by mishmanners** [comment]
Thanks so much for including the Release Radar in this.
